tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "sourceMap": true,
     "inlineSources": true,
     "strict": true,
-    "target": "es6"
+    "target": "es6",
+    "typeRoots": ["./node_modules/@types", "./typings"]
```

Comment:
It evidently doesn't cause any failures, but why _shouldn't_ we include `./src/@types` here?

Reply:
because typeRoots is for associating existing files with their associated types. the way we're using @types is to store shared/external types that we consume and use in our other files.

Reply:
for example when you install @types/react or @types/node you do not import those types -- you import things from node/react

Reply:
Other files as in our other repos/libraries? That makes sense!